connection.go

```diff
@@ -454,7 +454,7 @@ func (c *Connection) Send(message *iso8583.Message, options ...Option) (*iso8583
 	return resp, err
 }
 
-func (c *Connection) writeMessage(w io.Writer, message *iso8583.Message) error {
+func (c *Connection) writeMessage(_ io.Writer, message *iso8583.Message) error {
```

Comment:
Why is `w` passed in if it's not used?

Reply:
Thank you for pointing that out. I thought it was necessary to comply with the MessageWriter interface, so I decided that it would be better not to introduce any breaking changes, but that was a mistake. Originally, it seems that the argument `w` [was being passed to WriteMessage](https://github.com/moov-io/iso8583-connection/pull/55/files), which was being used internally.

Reply:
fixed: https://github.com/moov-io/iso8583-connection/pull/87/commits/ef898a3d4d3f226fcc598b2d81b7d3a8a11f7b2a